docs/motivation.md

```diff
@@ -130,7 +130,7 @@ can be triggered by an NOS-external application.
 
 ## Requirements
 
-* **[R1]** The elements that are provided as a dynamic parameter should be _in
+* **[R1]** The elements that are able to be changed more rapidly should be _in
```

Comment:
here and throughout, can we standardize on terms for the configuration stores:
volatile (as per gsii definition)
dynamic (as per bootz definition)
applied (as per the draft-openconfig-netmod-opstate-01)

So for example:

```suggestion
* **[R1]** The applied= set of configuration items is the union of `volatile` configuration items and  `dynamic` configuration items on the device.   Removing the volatile set should not affect the change the dynamic set.

* **[R2]** volatile configuration items override dynamic configuration items.
```

Reply:
Given that these requirements are described independently of the solution -- I don't think this makes sense to do. This document is meant to sketch out the initial solution and similarly to gRIBI, I propose to write a specification *after* iterating on the design.

This requirement as you have suggested is very specific to an individual solution.